pkg/driver/node_test.go

```diff
@@ -692,3 +753,18 @@ func getNodeMock(mockCtl *gomock.Controller, nodeName string, returnNode *corev1
 
 	return mockClient, mockNode
 }
+
+func expectErr(t *testing.T, actualErr error, expectedCode codes.Code) {
+	if actualErr == nil {
+		t.Fatalf("Expect error but got no error")
+	}
+
+	status, ok := status.FromError(actualErr)
+	if !ok {
+		t.Fatalf("Failed to get error status code from error: %v", actualErr)
+	}
+
+	if status.Code() != expectedCode {
+		t.Fatalf("Expected error code %d, got %d message %s", codes.InvalidArgument, status.Code(), status.Message())
```

Comment:
Wouldn't we expect `expectedCode` not `codes.InvalidArgument`?

Reply:
Good catch, thank you!